### ISSUE TITLE
No function with blogger

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#795]
```
Since last Friday (01-04-14) Scribe does'nt work with Blogger, Google's 
Blogging Service. Maybe an error in the API's?

Wordpress is ok. and others too

I use Firefox latest and Chrome latest

SribeFire Version is 4.3.2

Regards,

Axel B.



```

-----
Original issue reported on code.google.com by `hb374...@gmail.com` on 7 Jan 2014 at 10:18

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:01
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Great! It works! Thanks!!!!
```

-----
Original comment by `hb374...@gmail.com` on 15 Apr 2014 at 8:45
